source/_components/netatmo.markdown

```diff
@@ -24,6 +24,7 @@ netatmo:
   secret_key: YOUR_SECRET_KEY
   username: YOUR_USERNAME
   password: YOUR_PASSWORD
+  discovery: False
```

Comment:
Example code should always be minimized and since this in an optional variable it should not be in the example.
Feel free to have an advanced example also.